core/claim.go

```diff
@@ -353,19 +353,21 @@ type ClaimLinkObjectIdentity struct {
 	// IdAddr is the Ethereum Address related to the identity.
 	IdAddr common.Address
 	// ObjectHash is the hash of the object.
-	ObjectHash merkletree.Hash
+	ObjectHash [248 / 8]byte
 }
 
 // NewClaimLinkObjectIdentity returns a ClaimLinkObjectIdentity.
 func NewClaimLinkObjectIdentity(hashType uint32, objectType uint32, objectIndex uint16, idAddr common.Address,
```

Comment:
Could you define constant types for hashType and ObjectType?

Example:
```
// NodeType defines the type of node in the MT.
type NodeType byte

const (
	// NodeTypeMiddle indicates the type of middle Node that has children.
	NodeTypeMiddle NodeType = 0
	// NodeTypeLeaf indicates the type of a leaf Node that contains a claim.
	NodeTypeLeaf NodeType = 1
	// NodeTypeEmpty indicates the type of an empty Node.
	NodeTypeEmpty NodeType = 2
	// DBEntryTypeRoot indicates the type of a DB entry that indicates the current Root of a MerkleTree
	DBEntryTypeRoot NodeType = 3
)
```

Reply:
Both `hashType` and `ObjectType` defined